docs/content/install/shared-team-instance.md

```diff
@@ -187,4 +190,4 @@ Notice that the timestamps given have not changed.
 Next Steps:
 
 * Run the Athens Proxy on Kubernetes with Helm. [Coming Soon]
-* Explore best practices for running Athens in Production. [Coming Soon/Help Wanted](https://github.com/gomods/athens/issues/531)
\ No newline at end of file
+* Explore best practices for running Athens in Production. [Coming Soon/Help Wanted](https://github.com/gomods/athens/issues/531)
```

Comment:
I know this has nothing to do with your PR, but should this be https://github.com/gomods/athens/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22?

Reply:
I've flagged the linked issue with help wanted, and will fix up the link in a sec.